modules/luci-base/htdocs/luci-static/resources/cbi.js

```diff
@@ -630,8 +630,8 @@ String.prototype.format = function()
 						}
 
 						subst = (td > 0)
-							? String.format('%dd %dh %dm %ds', td, th, tm, ts)
-							: String.format('%dh %dm %ds', th, tm, ts);
+							? String.format('%d d %d h %d min %d s', td, th, tm, ts)
+							: String.format('%d h %d min %d s', th, tm, ts);
```

Comment:
What's the reason for extra spacing between the value and denominator?

Reply:
See my comment above.